pybliometrics/scopus/serial_title.py

```diff
@@ -13,35 +13,45 @@ def aggregation_type(self) -> str:
 
     @property
-    def citescoreyearinfolist(self) -> Optional[List[Tuple[int, float, None]]]:
-        """A list of two tuples of the form `(year, cite-score)`.  The first
-        tuple represents the current cite-score, the second tuple, if present,
-        represents the tracker cite-score.  For discontinued sources, there 
-        is no tracker cite-score, and the second tuple is just `None`.  See
-        https://service.elsevier.com/app/answers/detail/a_id/30562/supporthub/scopus/.
+    def citescoreyearinfolist(self) -> Optional[List[NamedTuple]]:
+        """A list of named tuples of the form `(year citescore status documentcount citationcount percentcited rank)`.
+        For a complete extraction of the data use the [view]() `'CITESCORE'``. The named tuple contains the following fields:
+        - `year`: Year
+        - `status`: Year 'In-progress' or 'Complete'
+        - `documentcount`: Number of documents of the last 4 years
+        - `citationcount`: Number of citations of the last 4 years
+        - `citescore`: citationCount/scholarlyOutput
+        - `percentcited`: Percent of documents cited
+        - `rank`: List of rank (and percentile) in different subjects
+
+        The `rank` is `None` or a named tuple of the form `(subjectcode rank percentile)`.
+
+        For more information check the [CiteScore documentation](https://service.elsevier.com/app/answers/detail/a_id/14880/supporthub/scopus/)
         """
-        def create_tuple(d, mode):
-            t = [d[f'citeScore{mode}Year'], d[f'citeScore{mode}']]
-            try:
-                t[0] = int(t[0])
-            except TypeError:  # year empty
-                return None
-            t[1] = float(t[1]) if t[1] else t[1]
-            return tuple(t)
-
+        
         try:
-            d = self._entry['citeScoreYearInfoList']
+            data = self._entry.get('citeScoreYearInfoList', {})
         except KeyError:
             return None
-        try:
-            current = create_tuple(d, "CurrentMetric")
-        except KeyError:
-            current = None
-        try:
-            tracker = create_tuple(d, "CurrentTracker")
-        except KeyError:
-            tracker = None
-        return [current, tracker]
+
+        # Named Tuples
+        info_fields = 'year citescore status documentcount '\
+            'citationcount percentcited '\
+            'rank'
+        rank_fields = 'subjectcode rank percentile'
+
+        named_info_list = namedtuple(
+            'Citescoreinfolist', info_fields, defaults=(None,) * len(info_fields.split()))
+        named_rank_list = namedtuple('Citescoresubjectrank', rank_fields)
+
+        # Extract depending on view
+        if self._view in ('STANDARD', 'ENHANCED'):
+            new_data = _get_two_cite_score_years(self, named_info_list, data)
+        elif self._view == 'CITESCORE':
+            new_data = _get_all_cite_score_years(
+                self, named_info_list, named_rank_list, data)
```

Comment:
Combine the lines such that the line break is after the comma of named_info_list